File: ___/_/Engineering/Software/Algorithms/euler001.c
/**
 * 001 <http://projecteuler.net>
 */
#include <stdio.h>

int main(void)
{
    int sum = 0;

    for (int i=1; i<1000; i++) {
        if (i%3 == 0 || i%5 == 0) {
            sum += i;
        }
    }
    printf("%d\n", sum);

    return 0;
}

File: source/project-euler/euler001.js
/* 
 * https://projecteuler.net/problem=1
 * execution: `$ node euler1.js`
 */

function euler1() {
    var sum = 0;
    for (var i = 3; i < 1000; ++i) {
        if (i % 3 === 0 || i % 5 === 0) {
            sum += i;
        }
    }
    return sum;
}

console.log(euler1());
